sso-samples/saml2-sso-sample/saml2-web-app-pickup-manager/src/main/webapp/home.jsp

```diff
@@ -379,39 +393,29 @@
                     <div class="col-md-6 d-block mx-auto">
                         <div class="card card-body table-container">
                             <div class="table-responsive content-table">
-                                <table class="table">
-                                    <%
-                                        if (saml2SSOAttributes != null && !saml2SSOAttributes.isEmpty()) {
-                                    %>
-                                    <thead>
-                                    <tr>
-                                        <th rowspan="2">User Details</th>
-                                    </tr>
-                                    </thead>
-
-                                    <tbody>
-
-                                    <%
-                                        for (Map.Entry<String, String> entry:saml2SSOAttributes.entrySet()) {
-                                    %>
-                                    <tr>
-                                        <td><%=entry.getKey()%></td>
-                                        <td><%=entry.getValue()%></td>
-                                    </tr>
-
-                                    <%
-                                        }
-                                    } else {
-                                    %>
-                                    <tr>
-                                        <p align="center"> Please configure claim mapping in SP configurations.</p>
-                                    </tr>
-                                    <%
-                                        }
-                                    %>
-                                    </tbody>
-                                </table>
-
+                                <% if (!subjectAttributeValueMap.isEmpty()) { %>
```

Comment:
Is there a possibility for `subjectAttributeValueMap` to be null ? If so add a null check